src/Migrations/2020/Version20200405160721.php

```diff
@@ -1,41 +0,0 @@
-<?php
-
```

Comment:
create a migration to revert the changes instead. Else this will result in many problem when deploying to the share